next.config.js

```diff
@@ -1,8 +1,5 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
-  experimental: {
```

Comment:
What is the reason of deleting this part? Deleting this throws an error on my side 👇🏽 

<img width="1125" alt="Screenshot 2023-10-26 at 08 06 45" src="https://github.com/YurisCodingClub/zero-minds/assets/54622834/bf950408-5cd9-45b2-8e0e-5c36970bdb5b">

Reply:
That's odd, the screenshot below shows my terminal when I leave the experimental flag in the next config. It gives a psuedo warning. Adding the experimental flag doesn't not affect the code though, it only throws that warning. I'll fix it back in

![Appdir](https://github.com/YurisCodingClub/zero-minds/assets/72811283/5db9119e-a782-4f85-ac0f-cb1e209b0aab)